README.md

```diff
@@ -52,7 +93,9 @@ Here are several advantages why you would want to use a relative import supporti
     Developers can directly use relative imports in their test code, resulting in a smoother transition to the
     testing framework.
 
-<h1></h1>
+<div align="right">[ <a href="#table-of-contents">↑ Back to top ↑</a> ]</div>
```

Comment:
Why is there back to top after each chapter?

Reply:
To get back to the top :)

These are just ideas, take any if you like, drop the rest.